client/gutenberg/editor/utils.js

```diff
@@ -12,17 +12,35 @@ import debugFactory from 'debug';
 
 const debug = debugFactory( 'calypso:gutenberg' );
 
-export const overrideAPIPaths = siteSlug => {
-	// TODO: no API support for now. We'll also need to handle authorization here.
-	apiFetch.use( () => noop );
+export const applyAPIMiddlewares = siteSlug => {
+	//make authenticated calls using the WordPress.com REST Proxy
+	//bypassing the apiFetch call that uses window.fetch
+	//first middleware in, last out
+	apiFetch.use( options => {
```

Comment:
Is it a problem that we never call `next` here? This likely breaks the middleware chain right?